src/Tizen.NUI/src/internal/Xaml/XamlParser.cs

```diff
@@ -319,15 +319,34 @@ static IValueNode GetValueNode(object value, XmlReader reader)
         static IList<XmlnsDefinitionAttribute> s_xmlnsDefinitions;
         public static IList<Assembly> s_assemblies = new List<Assembly>();// = new Assembly[]{};
 
-        static void GatherXmlnsDefinitionAttributes()
+        static void GatherXmlnsDefinitionAttributes(Assembly currentAssembly)
         {
             //this could be extended to look for [XmlnsDefinition] in all assemblies
             // var assemblies = new [] {
             // 	typeof(View).GetTypeInfo().Assembly,
             // 	//typeof(XamlLoader).GetTypeInfo().Assembly,
             // };
             // s_assemblies = new Assembly[]{typeof(View).GetTypeInfo().Assembly};
-            s_assemblies.Add(typeof(View).GetTypeInfo().Assembly);
+            //s_assemblies.Add(typeof(View).GetTypeInfo().Assembly);
+            var assemblies = currentAssembly?.GetReferencedAssemblies();
+
+            if (null == assemblies || 0 == assemblies.Length)
+            {
+                s_assemblies.Add(typeof(View).GetTypeInfo().Assembly);
+            }
+            else
+            {
+                foreach (var assembly in assemblies)
+                {
+                    try
+                    {
+                        s_assemblies.Add(Assembly.Load(assembly));
+                    }
```

Comment:
could you check the warning at line 345?
there is no action even though an exception occurs.
it seems not good. 
maybe it needs re-throw exception or give an error message.